src/strings/normalize.c

```diff
@@ -507,6 +521,8 @@ static MVMint32 is_spacing_mark(MVMThreadContext *tc, MVMCodepoint cp) {
     }
 }
 static MVMint32 should_break(MVMThreadContext *tc, MVMCodepoint a, MVMCodepoint b) {
+    int GCB_a = MVM_unicode_codepoint_get_property_int(tc, a, MVM_UNICODE_PROPERTY_GRAPHEME_CLUSTER_BREAK);
+    int GCB_b = MVM_unicode_codepoint_get_property_int(tc, b, MVM_UNICODE_PROPERTY_GRAPHEME_CLUSTER_BREAK);
```

Comment:
That can't be done, because both switches rely on the values of GCB_b and GCB_a (1/2 of them do).
I can also remove the hangul check once I add it to the switches, since we're already going to be checking GCB property so we can do it all at once.
I can also move `is_spacing_mark`, and that `is_grapheme_prepend`, since it's tested in the switches though for some reason would not work probably because of when we decide to call `should_break` is optimized for NFG_QC=Yes.